src/Core/Framework/Adapter/Twig/Node/MacroOverrideNode.php

```diff
@@ -0,0 +1,91 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\Adapter\Twig\Node;
+
+use Shopware\Core\Framework\Log\Package;
+use Twig\Attribute\YieldReady;
+use Twig\Compiler;
+use Twig\Node\CaptureNode;
+use Twig\Node\Expression\ArrayExpression;
+use Twig\Node\MacroNode;
+
+#[Package('framework')]
+#[YieldReady]
+/**
+ * @internal
+ * @deprecated tag:v6.8.0 - Will be removed, see SwMacroFunctionTokenParser
```

Comment:
this can not be removed, we still use it from the SwMacroFunctionParser -> that class will stay